src/input_cp2k_projection_rtp.F

```diff
@@ -78,7 +93,7 @@ SUBROUTINE create_projection_rtp_section(section)
 
       CALL keyword_create(keyword, __LOCATION__, name="REF_MO_INDEX", &
                           description="Indexes of the reference MO read from the .wfn reference file (see REF_MO_FILE_NAME). "// &
-                          "Use this keywork if REFERENCE_TYPE=SCF."// &
+                          "Use this keyword if REFERENCE_TYPE=SCF."// &
```

Comment:
Could you please also add a space after `REFERENCE_TYPE=SCF.` ?
Unfortunately, [my recent change](https://github.com/cp2k/cp2k/commit/7c90856a675bdfbc0d30dfa5c6a14958c10a4a2d) led to a merge conflicts.

Reply:
Done